src/openMVG/multiview/triangulation.cpp

```diff
@@ -94,12 +94,18 @@ void TriangulateL1Angular
   // Rf'0 = m'0 and f'1 = m'1
   // Eq. (11)
   const Vec3 z = mprime1.cross(mprime0);
-  const Vec3 xprime1 = t + (z.dot(t.cross(mprime1)) / z.squaredNorm()) * mprime0;
+  const double lambda0 = z.dot(t.cross(mprime1)) / z.squaredNorm();
```

Comment:
Since we see some code duplicated between multiple functions, I would factorize this code.

Reply:
ok, an inline function included into the .cpp file?

Reply:
We can start with inline in the cpp and see if we need to provide access to it in the header later on :+1:

Reply:
I push a proposal in the last commit. I also factorize the "Absolute to Relative" thing